go/vt/vtgate/evalengine/integration/testdata/mysql_golden_1645577591.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
This is not the right way to test the feature -- the `mysql_golden` files are supposed to be generated by our fuzzer. To properly test the new implementation, you need to add a test to our `comparison_test.go` file that tests the `FLOOR` call with many different inputs.

Here's an example testing a similar function call:

https://github.com/vitessio/vitess/blob/main/go/vt/vtgate/evalengine/integration/comparison_test.go#L471-L498